### PR TITLE
Footer linked up or cursor disabled

### PR DESCRIPTION
**Pull request by Aruzal**

### CODE REVIEW

File: frontend/src/components/AppFooter.vue [hunk 1]
```diff
@@ -37,14 +37,14 @@ export default {
         { content: 'Profile', href: '/profile' }
       ],
       info: [
-        { content: 'Privacy Policy', href: '/' },
-        { content: 'Terms of Use', href: '/' },
-        { content: 'Help', href: '/' },
-        { content: 'UNSW Handbook', href: '/' }
+        { content: 'Privacy Policy', href: '/privacy-policy' },
+        { content: 'Terms of Use', href: '/terms-of-service' },
+        { content: 'Help', href: '/', disabled: 'cursor: not-allowed;' },
```
Comment (DarkPurple141) [inline frontend/src/components/AppFooter.vue]: Just delete lol.

File: frontend/src/components/AppFooter.vue [hunk 2]
```diff
@@ -37,14 +37,14 @@ export default {
...
+        { content: 'UNSW Handbook', href: '/handbook' }
       ],
       social: [
-        { content: 'facebook', href: '/' },
-        { content: 'twitter', href: '/' }
+        { content: 'facebook', href: '/', disabled: 'cursor: not-allowed;' },
```
Comment (DarkPurple141) [inline frontend/src/components/AppFooter.vue]: link facebook to CSESoc.

Comment (DarkPurple141) [reply frontend/src/components/AppFooter.vue]: Add a Contribute link which points to the repo.

File: frontend/src/router/routes.js [hunk 3]
```diff
@@ -146,6 +146,12 @@ export default new Router({
       name: 'privacy-policy',
       component: () => import('../views/PrivacyPolicy')
     },
+    {
+      path: '/handbook',
+      name: 'handbook',
+      beforeEnter() { location.href = 'https://www.handbook.unsw.edu.au/' }
```
Comment (DarkPurple141) [inline frontend/src/router/routes.js]: ??? da faq i'ts an external link lol.

Comment (DarkPurple141) [reply frontend/src/router/routes.js]: This isn't a good solution. I know what you're trying to do.

File: frontend/src/components/AppFooter.vue (hunk 2 above)
Comment (NunoDasNeves) [reply frontend/src/components/AppFooter.vue]: imo don't link to CSEsoc lol... contribute link is a good idea

**Comment by NunoDasNeves** [comment]
in privacy policy, please change `<style>` to `<style scoped>` - at the moment it's applying to the whole page so the `<li>` elements in the footer are getting messed up!

**Pull request merged**